firestore-translate-text/functions/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "ES2020",
-    "lib": ["ES2020"],
+    "lib": ["ES2020", "dom"],
```

Comment:
Not certain that this is correct - however, I needed to add it to resolve some build issues that i was experiencing when trying to npm i on this repo.

Reply:
how can we figure out if it is correct?

Reply:
Some more context - without this, `npm run build` returns the following error: 
```
node_modules/@google-cloud/firestore/node_modules/google-gax/build/src/streamArrayParser.d.ts(27,23): error TS2304: Cannot find name 'AbortController'.
node_modules/@google-cloud/firestore/node_modules/google-gax/build/src/streamArrayParser.d.ts(28,19): error TS2304: Cannot find name 'AbortSignal'.
```

Reply:
An alternative solution would be to upgrade the `@types/node` package to a later version, for example:

 `"@types/node": "^18.11.9"`. 

It seems this is out of date since upgrading typescript [earlier this year](https://github.com/firebase/extensions/actions/runs/1918622451/jobs/2720983387).

Reply:
That seems like a better option...

Reply:
I saw you updated the @types/node, can we revert this change now?